<?php

namespace ContainerR6Dxx7V;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_YJtWrRhService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.yJtWrRh' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.yJtWrRh'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'avis' => ['privates', '.errored..service_locator.yJtWrRh.App\\Entity\\Avis', NULL, 'Cannot autowire service ".service_locator.yJtWrRh": it references class "App\\Entity\\Avis" but no such service exists.'],
            'entityManager' => ['services', 'doctrine.orm.default_entity_manager', 'getDoctrine_Orm_DefaultEntityManagerService', true],
            'validatorInterface' => ['privates', 'validator', 'getValidatorService', true],
        ], [
            'avis' => 'App\\Entity\\Avis',
            'entityManager' => '?',
            'validatorInterface' => '?',
        ]);
    }
}
